src/query/QueryResult.ts

```diff
@@ -0,0 +1,16 @@
+import MuseumNumber from 'fragmentarium/domain/MuseumNumber'
+
+type LemmaSequence = ReadonlyArray<ReadonlyArray<string>>
+
+export interface QueryItem {
+  id_: string
+  museumNumber: MuseumNumber
+  matchingLines: readonly number[]
+  matchCount: number
+  lemmaSequences?: readonly LemmaSequence[]
```

Comment:
Do we need this in frontend?

Reply:
It's not needed, not even in the backend (only used by MongoDB) so I'll remove it.